Steve Langasek <vorlon@debian.org>

Add to the blacklist certain HP laptop ACPI implementations that are
known to have problems under 2.6.18.  This patch is obsolete in 2.6.19
and above, where the kernel ACPI implementation has been fixed to handle
the oddities of these laptops.

Closes: #400488, #404143.

--- linux-2.6.orig/drivers/acpi/blacklist.c	2007-02-17 22:02:12.000000000 -0800
+++ linux-2.6/drivers/acpi/blacklist.c	2007-02-18 20:26:21.000000000 -0800
@@ -69,6 +69,12 @@
 	 "Incorrect _ADR", 1},
 	{"ASUS\0\0", "P2B-S   ", 0, ACPI_DSDT, all_versions,
 	 "Bogus PCI routing", 1},
+	/* HP nx6125, nx6325 */
+	{"HP    ", "SB400\0\0\0", 0x10000, ACPI_DSDT, all_versions,
+	 "Bogus fan support", 1},
+	/* HP nc6120 */
+	{"HP    ", "DAU00 \0\0", 0x10000, ACPI_DSDT, all_versions,
+	 "Bogus fan support", 1},
 
 	{""}
 };
